mysite/ct/api.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import viewsets
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.permissions import IsAdminUser
+
+from ct.serializers import ResponseSerializer, ErrorSerializer
+from ct.models import Response, StudentError
+
+
+class ResponseViewSet(viewsets.mixins.ListModelMixin, viewsets.GenericViewSet):
+    """
+    Django RestFramework class to implement Course student responses report.
+    """
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAdminUser,)
+    queryset = Response.objects.filter(kind='orct', unitLesson__order__isnull=False)
+    serializer_class = ResponseSerializer
+
+    def get_queryset(self):
+        queryset = super(ResponseViewSet, self).get_queryset()
+        course_id = self.kwargs.get('course_id')
```

Comment:
Instead of make a queryset, its enough to define a `lookup_field`. It will do the same job.